app/views/layout.erb

```diff
@@ -40,6 +40,7 @@
     <img src="/gorilla_banner.png" alt="gorilla banner" />
   </div>
       <div id="content">
+        <%= flash[:message] %>
```

Comment:
@jkrth617 I didn't think about it rendering nil all the time. That's a good point.